ios/Trifle/Sources/TrifleCertificate.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  TrifleCertificate.swift
+//  Trifle
+//
+
+import Foundation
+import Wire
+
+public class TrifleCertificate : Codable {
```

Comment:
does this need to be a codable? if so can we have a unit test similar to keyhandle for this?

Reply:
I can add unit test. It should be codable in case app wants to serialize/deserialize.